app/scripts/metamask-controller.js

```diff
@@ -2136,6 +2331,26 @@ export default class MetamaskController extends EventEmitter {
     return await promise;
   }
 
+  ///: BEGIN:ONLY_INCLUDE_IN(flask)
+  /**
+   * TODO:snaps verify safety of using keyringController.getAccounts
```

Comment:
What was the concern here exactly?

Reply:
You know, I can't recall. However, I have concerns about deriving app keys from the user's Ethereum addresses (we should just use some arbitrary derivation path instead probably), so I updated the docstring to reflect that. Also renamed the method to `getAppKeyForSubject`.